cmd/k8s-operator/ingress-for-pg.go

```diff
@@ -593,15 +586,8 @@ func (a *IngressPGReconciler) deleteVIPServiceIfExists(ctx context.Context, name
 	return nil
 }
 
-func (a *IngressPGReconciler) updateTailscaledConfigWithServices(ctx context.Context, pgName string, ingList *networkingv1.IngressList, logger *zap.SugaredLogger) error {
-	var servicesToAdvertise []string
-	for _, i := range ingList.Items {
-		if i.DeletionTimestamp.IsZero() && shouldExpose(&i) {
-			// TODO(tomhjp): Also check that it's ready.
-			servicesToAdvertise = append(servicesToAdvertise, hostnameForIngress(&i))
-		}
-	}
-	logger.Debugf("setting services to advertise: %q", servicesToAdvertise)
+func (a *IngressPGReconciler) maybeUpdateAdvertiseServicesConfig(ctx context.Context, pgName string, serviceName tailcfg.ServiceName, shouldBeAdvertised bool, logger *zap.SugaredLogger) (err error) {
+	logger.Debugf("will advertise %q: %v", serviceName, shouldBeAdvertised)
```

Comment:
is this meant to stay?

Reply:
Yeah I thought it was helpful at a debug level, but happy to remove it.

Reply:
I imagine it could be useful, maybe word it a bit more clearly?